app/assets/stylesheets/darkswarm/footer.scss

```diff
@@ -4,8 +4,8 @@
 
 footer {
   .row {
-    p a {
-      font-size: 0.875rem;
+    p a {  /* was p a  */
```

Comment:
Is there any reason to keep history as comment? I think `git` can do the same no?
_Edit:_ Ok, I saw the next commits ;)